resources/Materials/TestSuite/_options.mtlx

```diff
@@ -16,7 +16,7 @@
       <parameter name="overrideFiles" type="string" value="" />
 
       <!-- Comma separated list of light setup files. Only used for hardware shading languages  -->
-      <parameter name="lightFiles" type="string" value="default_viewer_lights.mtlx" />
+      <parameter name="lightFiles" type="string" value="" />
```

Comment:
Remove direct lights for now, since OSL testrender can't render with direct lights, and to compare results we need the same light setup.